cmd/glasskube/cmd/bootstrap.go

```diff
@@ -82,6 +75,43 @@ var bootstrapCmd = &cobra.Command{
 			}
 
 		}
+
+		upgradeNeeded := installedVersion != "" && semver.IsUpgradable(installedVersion, desiredVersion)
+		if !bootstrapCmdOptions.yes {
+			if upgradeNeeded {
+				if desiredVersion == "" {
+					confirmMessage := fmt.Sprintf("Glasskube is currently installed in this cluster (%s) "+
+						"in version %v. The version you are about to install is unknown. Please make sure the "+
+						"versions are compatible, this action could lead to a "+
+						"broken cluster!\nContinue?", currentContext, installedVersion)
+					if !cliutils.YesNoPrompt(confirmMessage, false) {
+						fmt.Println("Operation stopped")
+						cliutils.ExitWithError()
+					}
+				} else {
+					confirmMessage := fmt.Sprintf("Glasskube will be updated to version %s "+
+						"in cluster %s.\nContinue? ", desiredVersion, currentContext)
+					if !cliutils.YesNoPrompt(confirmMessage, true) {
+						fmt.Println("Operation stopped")
+						cliutils.ExitWithError()
+					}
+				}
+			} else if installedVersion[1:] == desiredVersion {
```

Comment:
`installedVersion` could be empty here, and then the `[1:]` access panics. So this condition should be `installedVersion != "" && ...`